### PR TITLE
Update Travis CI settings

### PR DESCRIPTION
**Pull request by sferik**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,37 @@
...
+  global:
+    # https://github.com/discourse/discourse/blob/master/.travis.yml
+    - RUBY_GC_MALLOC_LIMIT=50000000
+  matrix:
+    - AR_VERSION=3.1
+    - AR_VERSION=3.2
+    - AR_VERSION=4.0
+    - AR_VERSION=4.1
+    - AR_VERSION=4.2
+    - AR_VERSION=5.0
```
Comment (zdennis) [inline .travis.yml]: Very cool, didn't know this existed!

File: .travis.yml [hunk 2]
```diff
@@ -1,20 +1,37 @@
...
 script:
-  - test/travis/build.sh
+  - bundle exec rake test:mysql2
+  - bundle exec rake test:mysql2spatial
+  - bundle exec rake test:postgis
+  - bundle exec rake test:postgresql
+  - bundle exec rake test:seamless_database_pool
+  - bundle exec rake test:spatialite
+  # so far the version installed in travis seems < 3.7.11 so we cannot test sqlite3 on it
+  # bundle exec rake test:sqlite3
```
Comment (zdennis) [inline .travis.yml]: :*(

File: test/travis/build.sh [hunk 3]
```diff
@@ -1,34 +0,0 @@
...
-  run bundle exec rake test:postgresql              # Run tests for postgresql
-  run bundle exec rake test:seamless_database_pool  # Run tests for seamless_database_pool
-  run bundle exec rake test:spatialite              # Run tests for spatialite
-  # so far the version installed in travis seems < 3.7.11 so we cannot test sqlite3 on it
-  # run bundle exec rake test:sqlite3
-
-  #jruby
-  #bundle exec rake test:jdbcmysql               # Run tests for jdbcmysql
-  #bundle exec rake test:jdbcpostgresql          # Run tests for jdbcpostgresql
-done
```
Comment (zdennis) [inline test/travis/build.sh]: It was a good script while it lasted.

File: .travis.yml (hunk 2 above)
Comment (sferik) [inline .travis.yml]: Some discussion of the issue here: https://github.com/travis-ci/travis-ci/issues/3815.

Comment (sferik) [inline .travis.yml]: Fixed here: https://github.com/zdennis/activerecord-import/pull/258.

**Pull request merged**